src/desktop/apps/collections/server.js

```diff
@@ -29,13 +30,14 @@ app.get("/collections", async (req, res, next) => {
         styledComponents: true,
       },
       blocks: {
-        head: () => null,
+        head: () => <React.Fragment>{headTags}</React.Fragment>,
```

Comment:
@eessex @xtina-starr @l2succes FYI that it’s important to pull in the CSS defined by Reaction’s `Media` setup